Fix typos in "Instantiating a DateTimeOffset object"

GitHubPang opened the pull request.

GitHubPang committed:
Fix typos in "Instantiating a DateTimeOffset object"

IEvangelist commented:
Hi @GitHubPang - thanks for this contribution. Would you be able to add a _.vsproj_ as well to address the build issue?

https://github.com/dotnet/docs/runs/6203422671?check_suite_focus=true

> Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.

GitHubPang committed:
Add project.vb.vbproj.

File was copied from samples/snippets/visualbasic/VS_Snippets_CLR_System/system.idisposable/vb
